apps/cde-ui/src/app/pages/create-visualization-page/create-visualization-page.component.theme.scss

```diff
@@ -4,6 +4,7 @@
 @use '../../../styles/breakpoints' as breakpoints;
 
 $gray-palette: map.get(palettes.$palettes, 'gray');
```

Comment:
Move all the logic here into the regular scss file and use css variables instead